### ISSUE TITLE
First start failing 64bit, win10

### ISSUE THREAD

**Issue by dhrupal93** [jantje/arduino-eclipse-plugin#412]
When i first started, as shown below. V3
![capture1](https://cloud.githubusercontent.com/assets/11134572/14240889/cafb547a-fa65-11e5-8e51-3860eafdd347.PNG)
it shows following error:
![capture](https://cloud.githubusercontent.com/assets/11134572/14240868/ad0c39fc-fa65-11e5-9c6c-aee70c3af8e4.PNG)

And then shows:
![capture3](https://cloud.githubusercontent.com/assets/11134572/14240909/f201390e-fa65-11e5-86db-4e72ad385fbe.PNG)

on ignoring errors, it shows:
![capture2](https://cloud.githubusercontent.com/assets/11134572/14240979/81bbafc0-fa66-11e5-8e78-9a92af995204.PNG)

ERROR LOG: 
[eclipseError.pdf](https://github.com/jantje/arduino-eclipse-plugin/files/201931/eclipseError.pdf)

Note: Windows 10
EclipseArduino V3 (Downloaded on 3rd April 2016)
I have installed Java: 
![capture4](https://cloud.githubusercontent.com/assets/11134572/14241023/cd65e9b8-fa66-11e5-86b8-41e8af4ac621.PNG)

**Comment by jantje** [comment]
It looks like something went wrong when downloading the file http://downloads.arduino.cc/packages/package_index.json
Can you dump the content of [eclipse installation]/arduinoPlugin/package_index.json
After that delete this file and start the plugin again.

**Comment by rlogiacco** [comment]
@jantje should we consider to add a warning regarding the computer being connected and network configured (proxy wise) before starting the download process?

**Comment by wimjongman** [comment]
They just downloaded the plugin. I am fine with assuming that there is a network.

The Proxy is another thing.

**Comment by jantje** [comment]
on the  last image you see the welcome page which is also downloaded. http://eclipse.baeyens.it/product/V3_0.shtml
Looking at it again it looks like this is some special network setup/issue (there is a redirect warning on the 2'nd and 3th picture)
So I think this is a duplicate for #393 except for the file is redirected on an intermediate network server which sounds like a middleman network attack.
In other words I thin @dhrupal93 his computer is (intentionally?) hacked.

As to the proxy
AFAIK eclipse proxy setting is by default: take the default network settings. As this process is started at the very beginning, eclipse proxy settings should not interfere.

**Comment by dhrupal93** [comment]
Our campus has Proxy network, so when I installed with full eclipseArduino. it was not able to download the plugin after i opened for first time. Also it was asking for two .jason files (ESP and Arduino)

When I only opened eclipse (not eslipseArduino) . and then I dragged the plugin from market place. Now it works for eclipse with arduino.

**Comment by rlogiacco** [comment]
@dhrupal93 as you correctly guessed this has to do with the network configuration. Toddler tries to download all the necessary files on first start and a proxy will prevent that download to complete nicely.

@jantje if they set the network properties and restart, would the initial download attempted again?

**Comment by dhrupal93** [comment]
I tried that (open once, then set proxy and restart), but it was not working then also.

**Comment by jantje** [comment]
At startup Toddler checks if he has "boards". If no boards are found the default are downloaded.
A proxy should not cause issues, a "redirection" does.
Again I think @dhrupal93 his system is infected.

@dhrupal93 
Can you try on another machine on the same campus?

**Comment by jantje** [comment]
@dhrupal93 
Since there has not been any reaction in 25 days I'm closing this
